widget/embedded/src/containers/WidgetInfo/WidgetInfo.types.ts

```diff
@@ -13,7 +13,7 @@ export interface WidgetInfoContextInterface {
     details: ConnectedWallet[];
     totalBalance: string;
     isLoading: boolean;
-    refetch: (accounts: Wallet[], tokens: Token[]) => void;
```

Comment:
this can be a breaking change. i don't know why we'd got a second param and didn't use in implementation.